### PR TITLE
Fix broken link on license page

### PR DESCRIPTION
**Pull request by Alevsk**
Signed-off-by: Lenin Alevski <alevsk.8772@gmail.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/License/LicensePlans.tsx [hunk 1]
```diff
@@ -351,9 +351,7 @@ const LicensePlans = ({
                         onClick={(e) => {
                           e.preventDefault();
                           window.open(
-                            `${button.link}/?ref=${
-                              operatorMode ? "op" : "con"
-                            }`,
+                            `${button.link}?ref=${operatorMode ? "op" : "con"}`,
```
Comment (harshavardhana) [inline portal-ui/src/screens/Console/License/LicensePlans.tsx:354]: why not just have `con` for all calls?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/License/LicensePlans.tsx (hunk 1 above)
Comment (Alevsk) [reply portal-ui/src/screens/Console/License/LicensePlans.tsx:354]: I think we want to track when the click comes from the operator ui too

Comment (harshavardhana) [reply portal-ui/src/screens/Console/License/LicensePlans.tsx:354]: i thought we decided to do only one ref= @dvaldivia ?

Comment (dvaldivia) [reply portal-ui/src/screens/Console/License/LicensePlans.tsx:354]: I thought we decided to do two types of ref @harshavardhana xD

Comment (harshavardhana) [reply portal-ui/src/screens/Console/License/LicensePlans.tsx:354]: oh okay then

**Review by harshavardhana — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**